src/main/features/response/index.ts

```diff
@@ -36,9 +36,9 @@ export class Feature {
 
     app.all(allResponseRoutes, defendantResponseRequestHandler())
     app.all(allResponseRoutes, ClaimMiddleware.retrieveByExternalId)
-    app.all(/^\/case\/.+\/response\/(?!receipt|summary).*$/, IsDefendantInCaseGuard.check())
+    app.all(/^\/case\/.+\/response\/(?!receipt|summary|claim-details).*$/, IsDefendantInCaseGuard.check())
```

Comment:
A `claim-details` should be more within claim journey as there is nothing about response but I did not want to make too many unrelated changes. Something to cleanup in the future.